server/ws/subscriptions.go

```diff
@@ -456,32 +448,31 @@ func canForwardLiveEvent(ctx context.Context, filters model.Filters, in *model.E
 }
 
 func (h *handler) BroadcastNewEvents(ctx context.Context, events ...*model.Event) (numberOfSubscriptions int) {
-	h.Subscriptions.Range(func(_ string, sub subscription) bool {
-		authData, _ := h.ConnAuth.Load(sub.Writer)
-		for _, event := range events {
-			if !canForwardLiveEvent(ctx, sub.Source.Filters, event, &authData.UserDataContext) {
+	for _, event := range events {
+		for w, sub := range h.Subscriptions.Lookup(event) {
```

Comment:
Do we need any metrics for lookup to track this index speed or benchmarks is enough?